spec/robots/accession/technical_metadata_spec.rb

```diff
@@ -88,18 +126,99 @@
+              headers: {
+                'Content-Type' => 'application/json',
+                'Authorization' => 'Bearer rake-generate-token-me'
+              }
+            )
+            .to_return(status: 200, body: '', headers: {})
+        end
+
+        it 'file IS sent to techmd service' do
+          expect(perform.status).to eq('noop')
+        end
+      end
+
+      context 'when file has false for sdrPreserve' do
+        before do
+          unexpected_file = "file://#{workspace}/dd/116/zh/0343/dd116zh0343/content/folder1PuSu/story2r.txt"
+          stub_request(:post, 'https://dor-techmd-test.stanford.test/v1/technical-metadata')
+            .with(
+              body: hash_including(
+                files: array_including(a_string_matching(expected_file).and(excluding(unexpected_file)))
+              ),
+              headers: {
+                'Content-Type' => 'application/json',
+                'Authorization' => 'Bearer rake-generate-token-me'
+              }
+            )
+            .to_return(status: 200, body: '', headers: {})
         end
 
-        it 'does not run technical metadata' do
-          expect(perform.status).to eq('skipped')
+        it 'file is NOT sent to techmd service' do
+          expect(perform.status).to eq('noop')
         end
       end
+
+      context 'when file has no sdrPreserve setting' do
+        before do
+          unexpected_file = "file://#{workspace}/dd/116/zh/0343/dd116zh0343/content/folder1PuSu/story3m.txt"
+          stub_request(:post, 'https://dor-techmd-test.stanford.test/v1/technical-metadata')
+            .with(
+              body: hash_including(
+                files: array_including(a_string_matching(expected_file).and(excluding(unexpected_file)))
+              ),
+              headers: {
+                'Content-Type' => 'application/json',
+                'Authorization' => 'Bearer rake-generate-token-me'
+              }
+            )
+            .to_return(status: 200, body: '', headers: {})
+        end
+
+        it 'file is NOT sent to techmd service' do
+          expect(perform.status).to eq('noop')
+        end
+      end
+
+      context 'when file has no administrative metadata' do
+        before do
+          unexpected_file = "file://#{workspace}/dd/116/zh/0343/dd116zh0343/content/folder1PuSu/story4d.txt"
+          stub_request(:post, 'https://dor-techmd-test.stanford.test/v1/technical-metadata')
+            .with(
+              body: hash_including(
+                files: array_including(a_string_matching(expected_file).and(excluding(unexpected_file)))
+              ),
+              headers: {
+                'Content-Type' => 'application/json',
+                'Authorization' => 'Bearer rake-generate-token-me'
+              }
+            )
+            .to_return(status: 200, body: '', headers: {})
+        end
+
+        it 'file is NOT sent to techmd service' do
```

Comment:
per cocina models, which defaults to Administrative.new for File and that in turn gets preserve=false.